File: src/fizzbuzz.java
import java.util.Scanner;

public class fizzbuzz {

    public static void main(String[] args) {

//        boolean fizz = false;
//        boolean buzz = false;

        Scanner scanner = new Scanner(System.in);
        int x = scanner.nextInt();
        int y = scanner.nextInt();
        int n = scanner.nextInt();

        for (int i = 1; i <= n; i++) {

            if (i % x == 0 && i % y == 0) {

                System.out.println("FizzBuzz");
                //if both are divisible so we can print FIZZBUZZ
            }
            else if (i % x == 0) {

                System.out.println("Fizz");
                //do thing so we can print BUZZ
            }
            else if (i % y == 0) {

                System.out.println("Buzz");
                //do thing so we can print BUZZ
            }
            else {

                System.out.println(i);
            }

        }

    }
}

File: go/fizzbuzz.go
package main

import "fmt"

func main() {

	/*
		x = fizz divisor
		y = buzz divisor
		n = upper bound of range to print
	 */
	var x, y, n int

	fmt.Scanln(&x, &y, &n)

	for i := 1; i <= n; i++ {

		if (i % x == 0 && i % y == 0) {
			fmt.Println( "fizzbuzz")
		} else if (i % x == 0) {
			fmt.Println( "fizz")
		} else if (i % y == 0) {
			fmt.Println( "buzz")
		} else {
			fmt.Println(i)
		}

	}

}
